src/witness/yamlWitness.ml

```diff
@@ -287,7 +275,7 @@ struct
     let entries =
       if entry_type_enabled YamlWitnessType.LoopInvariant.entry_type then (
         LH.fold (fun loc ns acc ->
-            if WitnessInvariant.emit_loop_head && List.exists is_loop_head_node ns then (
+            if WitnessInvariant.emit_loop_head then ( (* TODO: remove double condition? *)
```

Comment:
What's up with the TODO?

Reply:
Stemming from the `loop_invariant`/`location_invariant` confusion times, we have an option `witness.invariant.loop-head` in addition to `witness.yaml.entry-types`. Currently both need to be enabled to get these invariants, but maybe the former option has been unnecessary for a while. This is something to be considered separately.